packages/models/package.json

```diff
@@ -142,6 +142,18 @@
       "module": "./dist/cohere/embedding.mjs",
       "require": "./dist/cohere/embedding.js"
     },
+    "./ollama/generation": {
```

Comment:
We need to explicitly export the types (above in `typesVersions`) otherwise editors and other type-checking code may break.